src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -222,14 +237,15 @@
      */
     private static final List<VCFInfoHeaderLine> ATTRS = CollectionUtil.makeList(
             new VCFInfoHeaderLine(ORIGINAL_CONTIG, 1, VCFHeaderLineType.String, "The name of the source contig/chromosome prior to liftover."),
-            new VCFInfoHeaderLine(ORIGINAL_START, 1, VCFHeaderLineType.String, "The position of the variant on the source contig prior to liftover.")
+            new VCFInfoHeaderLine(ORIGINAL_START, 1, VCFHeaderLineType.String, "The position of the variant on the source contig prior to liftover."),
+            new VCFInfoHeaderLine(ORIGINAL_ALLELES, VCFHeaderLineCount.R, VCFHeaderLineType.String, "A list of the original alleles (including REF) of the variant prior to liftover.")
             );
 
     private VariantContextWriter rejects;
     private final Log log = Log.getInstance(LiftoverVcf.class);
     private SortingCollection<VariantContext> sorter;
 
-    private long failedLiftover = 0, failedAlleleCheck = 0;
+    private long failedLiftover = 0, failedAlleleCheck = 0, totalRecoveredBySwapRefAlt = 0, totalNotRecoveredSwapRefAlt = 0;
```

Comment:
seems like these variables can be refactored into one variable, `nRefAltSwapVariants`, which can then be used in two different ways in the code depending on `RECOVER_SWAPPED_REF_ALT`.

Reply:
True, though it seemed clearer in the code to separate them as two variables, so the intent was clearer.  I renamed it to totalTrackedAsSwapRefAlt and merged them into one variable.